lms/templates/fields/field_link.underscore

```diff
@@ -1,11 +1,13 @@
-<label class="u-field-title" for="u-field-link-<%- id %>">
+<span class="u-field-title" for="u-field-link-<%- id %>">
     <%- gettext(title) %>
-</label>
+</span>
 <span class="u-field-value">
-    <a id="u-field-link-<%- id %>" href="<%- gettext(linkHref) %>" aria-describedby="u-field-message-<%- id %>">
-        <%- gettext(linkTitle) %>
+    <a id="u-field-link-<%- id %>" href="<%- gettext(linkHref) %>" aria-describedby="u-field-help-message-<%- id %>">
+       <span class="sr">gettext(screenReaderText)</span>
```

Comment:
I'm not sure what happened here.  This was fine as it was, at least in the case of the Reset Password link.  You don't want to ahve an aria-hidden on the line below.  Also, There is currently a bug here.  The "SR" text is not getting evaluated, so it reads "gettext(screenReaderText)".  In this case, there is no need for SR text at all since the Primary text is just fine.